app/models/author.rb

```diff
@@ -1,3 +1,5 @@
 class Author < ApplicationRecord
   has_and_belongs_to_many :papers
+  validates :name, uniqueness: { case_sensitive: false }, presence: true
+
```

Comment:
sorry, I have an undisciplined way of writing code.  I'll add and get rid of things and forget I had white space.